purchase_work_acceptance_invoice_plan/wizard/select_work_acceptance_invoice_plan_wizard.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
```

Comment:
```suggestion
from odoo.exceptions import UserError
```
Normally, ValidationError will be used for system error. I think what we have is just a warning message, so we can use UserError